scheduler/kubernetes/snaps-group.md

```diff
@@ -43,14 +46,14 @@ metadata:
   name: mysql-snapshot
   namespace: default
   annotations:
-    px/snapshot-type: group
-    tier: prod
-    type: db
+    portworx/snapshot-type: local
+    portworx/tier: prod
+    portworx/type: db
 spec:
   persistentVolumeClaimName: mysql-data
 ```
 
-*persistentVolumeClaimName* in the above spec is not relevant for group snapshots. It needs to be present in the spec since the schema requires that. The actual PVCs are actually selected using the annotations.
+*persistentVolumeClaimName* in the above spec can be name of any single PVC that will get matched using the selector or the group ID.
 
 Once you apply the above object you can check the status of the snapshots using `kubectl`:
```

Comment:
Can we also add a comment saying that if you delete the "volumesnapshots/mysql-snapshot" it will delete all the snaps in the group.

Reply:
done